app/assets/stylesheets/forever_style_guide/modules/_pnav.scss

```diff
@@ -0,0 +1,464 @@
+
+$f-pnav__main--zindex: 1030;
+$f-pnav__side--width: 320px;
+$f-pnav__side--zindex: 1032;
+
+@mixin deals_link_base() {
+  color: $color-forever_orange;
+
+  &:hover {
+    color: $color-forever_blue;
+  }
+
+  &::after {
+    position: relative;
+    top: -5px;
+    right: -2px;
+    font-family: FontAwesome;
+    font-size: 10px;
+    content: '\f005';
+  }
+}
+
+.f-pnav {
+  width: 100%;
+  padding: 0 10px;
+  box-shadow: 0 3px 5px rgba(0, 0, 0, 0.1);
+  background-color: $color-white;
+
+  /*support store method of affixing*/
+  &.affix {
+    top: 0;
+    left: 0;
+    z-index: $f-pnav__side--zindex;
+  }
+
+  /* nested to give more specificity and override the base styleguide link styles */
+  .f-pnav__link-base {
+    text-decoration: none;
+    color: $color-gray-600;
+    @include transition_fast(color);
+
+    &.is-active,
+    &:hover {
+      color: $color-forever_blue;
+    }
+  }
+}
+
+.f-pnav__main {
+  display: flex;
+  min-height: 50px;
```

Comment:
we should probably define this as a variable

Reply:
:+1: